packages/sanity/src/core/store/bundles/__workshop__/ReleaseForm.tsx

```diff
@@ -0,0 +1,135 @@
+import {COLOR_HUES} from '@sanity/color'
+import {CalendarIcon} from '@sanity/icons'
+import {
+  Box,
+  type ButtonTone,
+  Card,
+  Flex,
+  Select,
+  Stack,
+  Text,
+  TextArea,
+  TextInput,
+} from '@sanity/ui'
+
+import {Button} from '../../../../ui-components/button'
+import {type BundleDocument} from '../types'
+
+function toSlug(value: string): string {
```

Comment:
Not super important, but we already include [SpeakingURL](https://www.npmjs.com/package/speakingurl) with Studio for the default slugify function. It might be more robust than doing it ourselves.

Reply:
TIL, thanks. 
@RitaDias we should consider this once we merge the forms

Reply:
I'll update this on my PR just so we use it already 👍

Reply:
Added 👍 
Unrelated to this comment: but related to the Hues, I'll update the UI to use the hues instead of tones once I start using this store.

Thanks 🙏